features/facebookImport/src/model/importer.js

```diff
@@ -50,25 +50,71 @@ const dataImporters = [
     PostsImporter,
 ];
 
+class ImporterExecutionResult {
+    constructor(importer, status, executionTime) {
+        this._importer = importer;
+        this._status = status || createSuccessStatus();
+        this._executionTime = executionTime;
+    }
+
+    get importer() {
+        return this._importer;
+    }
+
+    get status() {
+        return this._status;
+    }
+
+    get executionTime() {
+        return this._executionTime;
+    }
+
+    _extractDataFromStatus(status) {
+        return {
+            name: status.name,
+            message: status.message,
+        };
+    }
+
+    get reportJsonData() {
+        return {
+            format: "v2",
+            importerName: this.importer.constructor.name,
+            executionTime: this.executionTime.toFixed(1),
+            status: Array.isArray(this.status)
```

Comment:
Before these statuses were flattened, which made analysing the results more difficult than it should.

Reply:
OK, I have a few nits here:
1. naming nit: `format` should maybe be `formatVersion` or any other hint that indicates this is a version.
2. number nit: possibly using no decimals would have the same info.
3. If we have a format, is there anywhere an ADR that specifies that format? If that's going to be consumed by polyPedia, do we have a shared document that says this?

Reply:
1. changed to `formatVersion`
2. changed to use 0 decimals 
3. For now we do not have any docs for it. Currently we treat reports as confidential user data that's only consumed by a dedicated analysis

Reply:
Thanks, and thanks. Even if it's confidential, we should have some spec somewhere... Even if it's not uploaded to polyPedia.